### PR TITLE
seems to be parallel

### PR DESCRIPTION
**Pull request by DaryaTarasowa**
I've checked the major routes and the related to them actions. If I understand correctly, async.parallel will run the actions in parallel. By the benchmarkings I have noticed neither significant improvements, nor (luckily) decrease of performance.

### CODE REVIEW

File: configs/routes.js [hunk 1]
```diff
@@ -247,7 +247,7 @@ export default {
         page: 'deck',
         handler: require('../components/Deck/Deck'),
         action: (context, payload, done) => {
-            async.series([
+            async.parallel([
```
Comment (kadevgraaf) [inline configs/routes.js]: @kprist @mmeimaris - is this a good change? Or is there a good reason it should be series instead of parallel?

**Comment by kadevgraaf** [comment]
Looks good. I approve.

File: configs/routes.js (hunk 1 above)
Comment (kprist) [reply configs/routes.js]: I think @stavmars or @huwf might have a better idea about this, they are updating stuff over in other branches that may make this PR obsolete

Comment (stavmars) [reply configs/routes.js]: @kadevgraaf this https://github.com/slidewiki/slidewiki-platform/pull/556 actually makes your change unnecessary.

**Comment by kprist** [comment]
Relevant code removed with https://github.com/slidewiki/slidewiki-platform/pull/556

**Pull request closed without merging**